src/components/cc-env-var-editor-simple/cc-env-var-editor-simple.js

```diff
@@ -139,8 +139,12 @@ export class CcEnvVarEditorSimple extends LitElement {
         display: none;
       }
 
+      /* Negative margin + padding to make the background-color span through the full width of the cc-block, despite the cc-block padding */
       cc-env-var-create {
-        margin-bottom: 1em;
+        background-color: var(--cc-color-bg-neutral);
+        margin-bottom: 0.5em;
+        margin-inline: -1em;
+        padding: 1em;
```

Comment:
The `overflow:hidden` of the `cc-expand` in the `cc-env-var-form` makes it tricky to obtain a full width background color for `cc-env-var-create`.
![image](https://user-images.githubusercontent.com/100240294/192303191-83450982-27d7-43e2-80e6-a3988ccf3968.png)